pontoon/base/templates/translate.html

```diff
@@ -171,6 +171,8 @@ <h2>Project info</h2>
             {{ Filter.item('all', 'All', count=True) }}
             {{ Filter.item('missing', 'Missing', count=True) }}
             {{ Filter.item('fuzzy', 'Fuzzy', count=True) }}
+            {{ Filter.item('warnings', 'Warnings') }}
```

Comment:
There are actually separate statuses, we just disabled them temporarily to avoid confusing users (numbers and colors would not added up ATM):
https://github.com/mozilla/pontoon/pull/1066/commits/11af3b86d9a383b601840609b121cf5b47c126ee

I think we should also hide the Errors and Warnings filters from the UI until [bug 1485611](https://bugzilla.mozilla.org/show_bug.cgi?id=1485611) is fixed.